Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -13,6 +13,7 @@
 #include <FormFolderBrowseEditWidget.h>
 #include <FormLineEditWidget.h>
 #include <FormLineEditTagsWidget.h>
+#include <FormOptionsWidget.h>
 #include <FormComboBoxWidget.h>
```

Comment:
because most of these are used as pointer member variables in this header, we should be forward declaring them here, and move these includes to the .cpp files at some point, doesn't have to be this review.